lib/test/info.test.js

```diff
@@ -4,6 +4,11 @@ test('valid german translation', () => {
   expect(info('treeType', 100).de).toStrictEqual('Tanne');
 });
 
+test('valid profile-specific german translation', () => {
+  // TODO: replace with real-life data
+  expect(info('slope', '<20', 'lu').de).toStrictEqual('kleiner 20 Prozent');
```

Comment:
Please replace this with some real LU data.

Reply:
Done